### DISCUSSION TITLE
Is this going to be continually updated?

### DISCUSSION THREAD

**Discussion by Science-geek42** [DeepGenX/CodeGenX#16] [General]
Hello, I just found this and think it's an amazing idea and I want to try using CodeGenX. However, I don't want to use it if this project will be forgotten quickly by the devs. So can I get a plan on what might be coming in the future or how long devs might stick to this project? I don't want to use this if the project ends up being dead and without updates/fixes in a month, you know.

**Comment by Aryagm** [comment]
Hi,

We are planning to stick to this project, and have no plans for dropping support or stopping development.
We believe this is the beginning and want to take this further. We are currently in the process of gathering additional data, and hope to fine-tune the model on a larger dataset for better performance.

Let me know if you have any other questions!

Thanks,
Arya

**Reply by Science-geek42** [reply]
Ok thanks very much for the help and quick reply I look forward to seeing how this project progresses.
